src/app/app.component.css

```diff
@@ -24,5 +24,23 @@
 
 /* NAVIGATION SUBMENU OVERRIDE EBI-GLOBAL TO REMOVE A WHITE RIGHT LINE */
 .no-border-submenu {
-  border-right: 0 !important ;
+  border-right: 0 !important;
 }
+
+
+.sticky-nav-container {
+  --global-margin: calc((100vw - 80rem) / -2);
+  position: sticky;
+  top: 0;
+  background: #007c82;
+  margin-left: var(--global-margin);
```

Comment:
There is a problem with this. Normally, this left margin is a negative value, which makes the container to move all the way to the left, but if the window is too small, it ends up being a positive value and pushes the container to the right, creating a weird empty space on the left.

This can be fixed by using `min`:
```
margin-left: min(var(--global-margin), 0px);
```

Check the video in https://drive.google.com/file/d/1DuU0VFE-9pbvfuXSmgO0aQEM3f4MnJIn/view?usp=drive_link to see how it currently behaves when you resize the window. This does not happen after the change suggested. I tried to attach the video here but I couldn't, as it is too big.

Reply:
Already done